homeworks/11/Domashno/src/main/java/com/example/domashno/UploadPicture.java

```diff
@@ -0,0 +1,59 @@
+package com.example.domashno;
+
+import jakarta.servlet.*;
+import jakarta.servlet.http.*;
+import jakarta.servlet.annotation.*;
+
+import java.io.IOException;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+
+@WebServlet(name = "UploadPicture", value = "/UploadPicture")
+public class UploadPicture extends HttpServlet {

```

Comment:
In the description of the problem such path doesn't exist. I suggest uniting this and GetPosts.java file and naming it PostsServlet as it deals with the posts